apps/osim/workflows/default.yml

```diff
@@ -1,34 +1,59 @@
 ---
 name: DEFAULT
-description: default vulnerability analysis
+description: default vulnerability workflow.
```

Comment:
nitpick: I am sorry for being constantly nitpicky about this but it irritates me a bit when there is a dot and no starting uppercase letter :smile: I would either use both or none. This is no sentence anyway so really up to you. And feel free to ignore my unnecessary nitpicks...

Reply:
Sorry about that, fixed.